src/main/java/ca/gc/aafc/dinauser/api/repository/UserRepository.java

```diff
@@ -26,29 +43,12 @@ public UserRepository() {
   }
 
   @Override
-  public DinaUserDto findOne(String id, QuerySpec querySpec) {
-    return service.getUser(id);
+  public DinaUserDto findOne(Serializable id, QuerySpec querySpec) {
+    return service.getUser(id.toString());
```

Comment:
Since DinaUserService has its own findOne with class validation, I think this method should use that instead of the bare .getUser.

Reply:
That makes sense thanks!